metricflow/model/objects/elements/measure.py

```diff
@@ -14,7 +14,9 @@ class NonAdditiveDimensionParameters(HashableBaseModel):
     """
 
     name: str
-    window_choice: AggregationType
+
+    # Optional Fields
+    window_choice: AggregationType = AggregationType.MIN
```

Comment:
I talked with Nick here, we wanted to default to MIN simply because of how our granularity works and the preference of a minimal config. For example, if we did `ds__week` and say we get 3 rows of `2022-01-01`, `2022-01-08`, `2022-01-15` if we had a MIN default, the result would look like the following,
```
2022-01-01  | value at 2022-01-01
2022-01-08  | value at 2022-01-08
2022-01-15  | value at 2022-01-15
```
If we did max, the result would look like this 
```
2022-01-01  | value at 2022-01-07
2022-01-08  | value at 2022-01-14
2022-01-15  | value at 2022-01-21
```
so intuitively it makes more sense to default as MIN rather than MAX. Open for further discussion here though or we could just simply keep this as required until we decide which is better.


As for the `AggregationType`, I agree with it being weird, but I was going to add a validation on that to ensure it's `MIN/MAX`.  I was thinking that would be enough and makes sure it gets blocked on usage of other types.

Reply:
The validation is fine for now, but a separate enum helps us out more, I think, because we can rely on the typechecker (and then the validation comes along with the JSON schema update).

The MIN thing makes sense, it's worth documenting that reasoning somewhere. I think the user-facing docs are the right place, actually, as long as it's not too in the weeds.